api/src/main/resources/application.yml

```diff
@@ -6,10 +6,15 @@ management:
     enabled: ${MANAGEMENT_SECURITY_ENABLED:true}
 
 auth.provider.service:
+  testing-support.enabled: ${AUTH_PROVIDER_SERVICE_TESTING_SUPPORT_ENABLED:}
```

Comment:
let's give this default value 
`testing-support.enabled: ${AUTH_PROVIDER_SERVICE_TESTING_SUPPORT_ENABLED:false}`